ansible-ipi-install/roles/installer/tasks/25c_disconnected_registry_create.yml

```diff
@@ -0,0 +1,186 @@
+---
+
+- name: Create a disconnected registry
+  block:
+    - name: Make sure needed packages are installed
+      package:
+        state: present
+        name: "{{ item }}"
+      loop:
+        - "{{ packages_registry | default([]) }}"
+      become: true
```

Comment:
so this we will want to move into the 30_packages.yml and have a when clause of when: inventory_hostname in groups['registry_host'] . You can leave this as is for now but will want to make that change before we merge in. Also moving the packages_registry from installer/vars to node-prep/vars

Reply:
also we want to confirm all those packages are actually needed.